docs/guides/core-concepts/introduction-to-cypress.mdx

```diff
@@ -1160,7 +1142,7 @@ they can fail, typically by passing a `{force: true}` option.
 
 ```js
 cy
```

Comment:
above where it says "all dom commands automatically wait" should that be queries?

Reply:
No; there are commands that interact with the DOM (like `.click()`) that aren't queries which wait, and queries (like `.readFile()`) which aren't related to the DOM at all.